### PR TITLE
Fixes CheckOV findings related to GuardDuty

### PR DESCRIPTION
**Pull request by dennisbrouwer91**
We need to explicitly enable the `guardduty_detector` resource.

### CODE REVIEW

File: audit.tf [hunk 1]
```diff
@@ -122,6 +123,12 @@ resource "aws_guardduty_detector" "audit" {
   }
 }
 
+resource "aws_guardduty_organization_configuration" "audit" {
+  auto_enable = true
+  detector_id = aws_guardduty_detector.audit[0].id
+}
+
+
```
Comment (shoekstra) [inline audit.tf:131]:
```suggestion

```

Can you pls remove the double line here.

**Review by shoekstra — CHANGES_REQUESTED** [review]
Thanks for this @dennisbrouwer91! Small change to do, and can you add an entry to the changelog please?

File: audit.tf (hunk 1 above)
Comment (dennisbrouwer91) [reply audit.tf]: Fixed!

**Review by sbkg0002 — DISMISSED** [review]
Readme? Fine!

**Review by shoekstra — APPROVED** [review]

**Pull request merged**